vision.md

```diff
@@ -1,115 +1,407 @@
+      the framework
+
+### Capabilities
+
+<!--
+The following list of capabilities should be considered non-final. It might be
+extended in the future.
+-->
+
+In combination with IoT platforms, thin-edge.io is a foundation for enabling
+devices with the following capabilities:
+
+1. Connectivity to the major cloud providers in the IIOT space
+    - for effortless and secure edge device lifecycle management for single
+      and device fleets
+    - for low-touch provisioning of thin edge devices
+    - for local and remote configuration
+    - for local and remote maintenance including remote access
+      (monitoring/troubleshooting)
+    - decommissioning of thin devices (e.g. for security compromised or end of
+      life devices)
+1. "South bound" connectivity to devices via the major protocols in the IIOT industry
+1. On-device data preprocessing
+    1. Analytical (timeseries analysis, ML, etc...)
+    2. Mathematical operations (avg, sum, etc...)
+    3. Logical operations
+1. On-device data generation (e.g. generating of events)
+1. Device management functionality
+    1. OS updates
+    1. Package updates
+    1. Configuration updates (Of both the thin-edge.io deployment as well as other software)
+    1. Firmware updates for "south bound"/child devices
+1. Device interaction
+    1. File upload/download
+    1. Filesystem listings
+    1. General system informations
+1. Persist data as given by components in different databases (e.g. PostgreSQL, Sqlite, MongoDB, Redis, Memcached, ...)
+1. Documentation of each component and their configuration
+    - This includes information on how to configure each configurable aspect of the component and its valid states
+    - This also includes all message types that the framework knows about
+    - Users with custom thin-edge.io deployments must be able to generate such a documentation themselves
+
+## Closing words
+
+thin-edge.io offers a unique approach to unify the needs of both the IT and OT
+world by offering a platform design focused on efficiency robustness and
+security while offering the highest level of extensibility.
+
+- We are not restricting users towards one specific software artifact type,
+  package manager, programming language or message payload to be used on the
+  device
+- We combine robust and lightweight components with extensibility (plug-in
+  mechanisms, mapper concept for cloud/platform support)
+- We offer out-of-the-box modules to be used in combination with device
+  management platforms
+- We offer hardware and infrastructure agnostic deployment of all edge
+  capabilities
 
-What business goals do we try to achieve ?
-- This is an open source project licenced under Apache 2.0 , it is focused on enterprise and company (B2B) use, hence why we encourage further partners to join our mission and community to create an industry standard for IoT device enablement and vendor agnostic connectivity and device management. 
```

Comment:
I feel the community aspect is a little bit lost here. We have quite a lot to achive and it would be great to call out we can only do tat by expanding our community , "divide and conquer" , and by also be open to collaborate and include exiting framworks to give our users a good package of features